phpBB/phpbb/textformatter/s9e/factory.php

```diff
@@ -126,6 +132,20 @@ class factory implements \phpbb\textformatter\cache_interface
 			</xsl:attribute>
 			<xsl:apply-templates/>
 		</a>',
+		'mention' => '<xsl:text>@</xsl:text>
+		<xsl:choose>
+			<xsl:when test="@profile_url">
+				<a class="mention">
```

Comment:
Not that it makes a practical difference but you can set the `href` attribute directly on the element:
```xsl
<a class="mention" href="{@profile_url}">
```